app/main.py

```diff
@@ -1,11 +1,11 @@
 class Distance:
-    def __init__(self, km):
+    def __init__(self, km: (int, float)) -> None:
```

Comment:
These annotations will not work unfortunately :(
Use either one of these instead
if python 3.10+ use: int | float
if older : Union[int, float]